android-components/components/service/nimbus/messaging.fml.yaml

```diff
@@ -90,15 +92,19 @@ objects:
       and call to action.
     fields:
       action:
-        # We'd like this to be ActionName, but actions can be https:// URLs,
-        # not just limited to keys in the actions map.
-        # This would be a breaking change.
-        type: Text
+        type: ActionName
         description: >
           A URL of a page or a deeplink.
           This may have substitution variables in.
         # This should never be defaulted.
-        default: ""
+        default: OPEN_URL
+      action-params:
+          description: >
+            A string map containing query parameters that will be appended to the action URL.
+            This is useful for opening URLs in tabs, or specifying that the tab should be private.
+            The values will may have substitutions, e.g. "url": "https://example.com/id={uuid}"
```

Comment:
is it possible to include another example to give an idea of other possible string maps?